client/src/components/Button/Button.css

```diff
@@ -15,47 +15,47 @@
 .button:hover {
```

Comment:
This looks great! Can we change all the bs to uppercase for consistency?

.Button:hover